fastlane/lib/fastlane.rb

```diff
@@ -19,6 +19,7 @@
 require 'fastlane/documentation/docs_generator'
 require 'fastlane/other_action'
 require 'fastlane/plugin_manager'
+require 'fastlane/plugins/plugins'
```

Comment:
Looking at the directory structure it seems like we want all plugin related code in that folder, right? So we're gonna move the `plugin_manager`?